src/tests/system/tests/test_sssctl.py

```diff
@@ -2,11 +2,15 @@
 SSSCTL tests.
 
 :requirement: IDM-SSSD-REQ: Status utility
+:requirement: IDM-SSSD-REQ: Configuration validation
```

Comment:
I am not sure that multiple requirements for one test will work properly (probably will not).
The requirements need to be moved to test level.